src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -418,9 +418,7 @@ public static String jobConfig(String jobKey, String cfgName) {
             this.put(ACTIVATION_KEY_AUTHENTICATION, "true");
 
             this.put(AUTH_OVER_HTTP, "false");
-            // By default, environments should be hidden so clients do not need to
-            // submit one when registering.
```

Comment:
This comment worries me. Have we vetted what modern subman does in this case? While environments are supported in hosted, they aren't *required*; so we need to make sure that flow isn't busted before we can change this. Probably.